spec.md

```diff
@@ -77,6 +77,23 @@ on their placement within source code.
 function crashTheSystem() {/* … */}
 ```
 
+dx supports all [JavaScript comment styles]: multi-line comments, single-line comments,
+and with or without leading `*` marks in each row. The following are equivalent:
+
+```js
+// Crashes the system
```

Comment:
Maybe "crashes the system" is confusing when you're saying all these comment styles are valid

Reply:
Haha, yeah - I'll chose a less intense example